packages/react-core/src/components/Pagination/PaginationOptionsMenu.tsx

```diff
@@ -26,7 +26,7 @@ export interface PaginationOptionsMenuProps extends React.HTMLProps<HTMLDivEleme
   perPageSuffix?: string;
   /** The type or title of the items being paginated */
   itemsTitle?: string;
-  /** The text to be displayed on the Options Toggle */
+  /** Accessible label for the Options Toggle */
```

Comment:
I just spot checked a handful of other react component props and we use the "accessible label" wording to describe the function of a `[whatever]AriaLabel` prop. This may be OK, just wanted to mention it.

Reply:
That's a fair point. My main reason for changing the description was due to "text to be displayed" didn't seem accurate either, as it seemed like the prop was only being used to add an `aria-label` to that dropdown toggle.